driver/test/java/io/crate/client/jdbc/integrationtests/CrateJDBCConnectionTest.java

```diff
@@ -129,8 +129,14 @@ private static void insertIntoTable() throws SQLException {
         preparedStatement.setObject(10, objectField);
         preparedStatement.setString(11, "1970-01-01");
         preparedStatement.setString(12, "127.0.0.1");
-        preparedStatement.setArray(13, connection.createArrayOf("varchar", new String[]{"a", "b", "c", "d"})); // TODO array support
-        preparedStatement.setArray(14, null);
+        preparedStatement.setArray(13, connection.createArrayOf("varchar", new String[]{"a", "b", "c", "d"}));
```

Comment:
on (1) it turns out that the TypeInfoCache already has a mapping/alias function enabled that already works for some Crate types (like integer -> int4) - think this would suffice?  https://github.com/crate/pgjdbc/blob/crate/pgjdbc/src/main/java/org/postgresql/jdbc/TypeInfoCache.java#L107 We could map, say, 'string' -> 'varchar' this way and it works properly, without the need for an extra function